### PR TITLE
created simple add name form

### PR DESCRIPTION
**Pull request by mspremulli**
used React hooks for now. We can switch to Redux later if we want

### CODE REVIEW

File: src/components/NameBox.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React, {useState , useEffect} from 'react'
+
+export const NameBox = () => {
+  const [name, setName] = useState("");
+  useEffect(() => {
```
Comment (jsartin513) [inline src/components/NameBox.js:5]: Does this do anything as it's defined now?

File: src/components/NameBox.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    
+  }, []);
+
+  const onChange = (e) => {
+    console.log(name);
```
Comment (jsartin513) [inline src/components/NameBox.js:10]: At some point we'll need to take this out for cleanliness

**Review by jsartin513 — APPROVED** [review]

File: src/components/NameBox.js (hunk 1 above)
Comment (mspremulli) [reply src/components/NameBox.js:5]: not yet. we can add it later if we want to make something happen when the component loads for the first time.

File: src/components/NameBox.js (hunk 2 above)
Comment (mspremulli) [reply src/components/NameBox.js:10]: yup

**Pull request merged**